modules/openapi-generator/src/main/resources/python/python-experimental/setup.mustache

```diff
@@ -23,6 +23,10 @@ REQUIRES.append("aiohttp >= 3.0.0")
 {{#tornado}}
 REQUIRES.append("tornado>=4.2,<5")
 {{/tornado}}
+{{#hasHttpSignatureMethods}}
```

Comment:
Optionally, why not spreading out each requirement to a separate line.
Then we don't need to use append statement.
```
REQUIRES = [
    "urllib3 >= 1.15",
    ...
{{#asyncio}}
    "aiohttp >= 3.0.0"
{{/asyncio}}
```

Reply:
done.